### DISCUSSION TITLE
Problem about example 1

### DISCUSSION THREAD

**Discussion by haimingxie** [ORTools4MTEX/ORTools#55] [Q&A] [answered]
Dear Doctor,

I hope this message finds you well. I want to express my sincere gratitude for sharing the ORTOOLS. I find this tool incredibly useful. However, I encountered some issues while trying to run Example 1 in MATLAB. Specifically, I encountered an error during the "reconstruct parent microstructure" step. The main problem is that the program cannot perform the assignment operation because the size of the left matrix is 95033, while the size of the right matrix is 950*9. Could you please provide some suggestions on how to resolve this issue? If you have the time, could you assist me in solving this problem? I am immensely thankful for your help.

Best wishes..

**Comment by AzdiarGazder** [comment]
HI @haimingxie,

I just ran Example 1 using the developer versions of both, MTEX 5.10.2 and ORTools (with MATLAB 2021b).
I'm afraid the script ran perfectly without any errors.
I could not replicate your issue.

I suggest you use the developer versions of both, MTEX 5.10.2 and ORTools as well.
In case you do run into errors, please post the complete error in the MATLAB command window in your discussion.
You can also post parts of the script where the error is happening as well as any other screenshots if needed.
Lastly, if you are using your own ebsd maps with any of the examples and running into an error, please send us the map as well.
All of the above information helps us diagnose where and why you are facing a problem.

Warm regards,
Azdi

**Reply by haimingxie** [reply, marked as answer]
Thank you very much for your generous answer, my problem was solved after changing the version of the tool. Thank you again for sharing the ORS tool.Best wishes.
